tests/integration/targets/aci_access_span_dst_group/tasks/main.yml

```diff
@@ -0,0 +1,323 @@
+# Test code for the ACI modules
+# Copyright: (c) 2023, Akini Ross (akinross@cisco.com)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
+    msg: 'Please define the following variables: aci_hostname, aci_username and aci_password.'
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+- name: Set vars
+  ansible.builtin.set_fact:
+    aci_info: &aci_info
+      host: "{{ aci_hostname }}"
+      username: "{{ aci_username }}"
+      password: "{{ aci_password }}"
+      validate_certs: '{{ aci_validate_certs | default(false) }}'
+      use_ssl: '{{ aci_use_ssl | default(true) }}'
+      use_proxy: '{{ aci_use_proxy | default(true) }}'
+      output_level: '{{ aci_output_level | default("info") }}'
+
+# CLEAN TEST ENVIRONMENT
+
+- name: Query all span ansible_group
+  cisco.aci.aci_access_span_dst_group:
+    <<: *aci_info
+    state: query
+  register: query_for_clean
+
+- name: Clean access span destination groups
+  cisco.aci.aci_access_span_dst_group:
+    <<: *aci_info
+    destination_group: "{{ item.spanDestGrp.attributes.name }}"
+    state: absent
+  loop: "{{ query_for_clean.current }}"
```

Comment:
This is good. We should add a file, that runs first which cleans the environment at the start of CI/CD